cicd/gitops/health-information-for-resources-deployment.adoc

```diff
@@ -6,6 +6,10 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-The environment details page displays the health status of the application resources, such as routes, synchronization status, deployment configuration and deployment history.
+The {gitops-title} *Environments* page in the *Developer* perspective of the {product-title} web console shows a list of the successful deployments of the application environments, along with links to the revision for each deployment.
+
+The *Application environments* page in the *Developer* perspective of the {product-title} web console displays the health status of the application resources, such as routes, synchronization status, deployment configuration, and deployment history.
+
+The *Application environments* pages in the *Developer* perspective of the {product-title} web console are decoupled from the {gitops-title} Application Manager command-line interface (CLI), `kam`. You do not have to use `kam` to generate the Application Environment manifests in order for the environments to show up in the *Developer* perspective of the {product-title} web console. You can use your own manifests, but the environments must however still be represented by namespaces. In addition, specific labels and annotations are still needed.
```

Comment:
```suggestion
The *Environments* pages in the *Developer* perspective of the {product-title} web console is decoupled from the {gitops-title} Application Manager command-line interface (CLI), `kam`. You do not have to use `kam` to generate the Application Environment manifests in order for the environments to show up in the *Developer* perspective of the {product-title} web console. You can use your own manifests, but the environments must however still be represented by namespaces. In addition, specific labels and annotations are still needed.
```
@ciiay correct me if I am wrong

Reply:
@varshab1210 I did discuss this with Yi Cai. It seems that the **Environments** pages were decoupled earlier from the `kam` CLI. But as I observe, I do not see this fact mentioned anywhere in the previous versions of GitOps RN or in the OCP GitOps documentation

Now, with v1.8, even the *Application environments* pages are decoupled from the `kam` CLI.

Hence to avoid confusion, I mentioned both the names of the pages in the paragraph. Later, Yi Cai corrected me to include only the *Application environments* pages in the paragraph.

@ciiay: Please review the paragraph and the confusion that it creates and provide your suggestion. Like Varsha, even I had the same question and confusion. Kindly provide guidance. Thank you!

Reply:
@varshab1210 Before this enhancement, the Environments page shows apps/envs which are not generated by KAM. And when you click into the Application environments page of the non-KAM env, it's not loading correctly. Now with this enhancement, the Application environments page works correctly as KAM generated envs as well. Not sure the best way to describe it but Environments page are not the point of this update from my understanding.

Reply:
@varshab1210 I get that by *Environments* pages you meant both `Environments` page and `Application environments` page. My suggestion is to use `Application environments page` so that it's only about one page. We can also mention that Environments page are decoupled from KAM in earlier version to avoid the confusion. 

```
The *Application environments* page in the *Developer* perspective of the {product-title} web console is decoupled from the {gitops-title} Application Manager command-line interface (CLI), `kam`. ...
```

Reply:
@ciiay Please let me know in which version did the Environments page got decoupled from KAM.

Reply:
Resolving this as per the discussion. There is no need to mention the pages explicitly.